.eslintrc.js

```diff
@@ -1,24 +1,29 @@
 module.exports = {
+  parser: "@typescript-eslint/parser",
   env: {
     browser: true,
     commonjs: true,
-    es6: true,
+    es2020: true,
```

Comment:
Is this the cool new thing that includes two space indentation and no-semi colons?

Reply:
No, this is needed for native `BigInt` support with the linter.

As far as I know, there is no "cool new thing" for indentation and semi-colons. While the large `.eslintrc` configuration file got removed, it was being ignore anyway and all of the configuration in there wasn't even being read. As a test I went back a couple commits and changed the `.eslintrc` file to be:
```
{
  asdofiadsi alfdoijalsidfh asdfli
```
And ran `yarn lint` and got no errors. (doing the same thing with the `.eslintrc.js` file  gives me syntax errors when I try to run the linter).

Reply:
I just checked, and `.eslintrc` without an extension has been deprecated as far back as 2018. Since both `.eslintrc` and `.eslintrc.js` existed when `.eslintrc` was [first added in 2019](https://github.com/gnosis/dex-contracts/tree/cb6b01ffb8a1cd31c0433a19ccfb27acc4edfd6a), the configuration in `.eslintrc`without an extension has always been ignored by our CI and never actually used.

It is possible that your editor prefers `.eslintrc` over `.eslintrc.js`, but that is not standard and not what our CI was doing anyway, so any rules in the `.eslintrc` without an extension have never actually been verified on our CI.